docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  db:
+    image: "postgres"
+    restart: unless-stopped
+    networks:
+      - "mylocal"
+    env_file:
+      - ".env"
```

Comment:
The same environment file goes to both containers, do both of them need all the data in the file?

Reply:
Yes, migrator needs parameters to connect to database

Reply:
Does it need all parameters?